docs/DeveloperGuide.md

```diff
@@ -476,20 +476,333 @@ testers are expected to do more *exploratory* testing.
    2. Re-launch the app using the `java -jar contactsforgood.jar` command.<br>
        Expected: The most recent window size and location is retained.
 
+### Testing the empty ContactsForGood Application
+**Purpose:** To ensure that commands can be executed in an empty state without crashing the application.
+1. To begin, reset the application state to an empty state by using the `clear` command.
+   1. Test case: `list`  
+      **Expected:** "Listed all persons" should be printed. 
+      The area below the output should not have any contacts displayed.
+   
+   2. Test case: `listGroup`  
+      **Expected:** "Listed all groups" should be printed.
+      The area below the output should not have any groups displayed.
+   
+   3. Test case: `delete 1`  
+      **Expected:** The error message "One or more indices provided are invalid" should be printed.
+   
+   4. Test case: `edit 1 n/John`  
+      **Expected:** The error message "One or more indices provided are invalid" should be printed.
+   
+   5. Test case: `search n/John`   
+      **Expected:** "0 persons listed!" should be printed.
```

Comment:
Have updated it such that it says "0 person listed!"

Reply:
Need change this again